docs/xdr/features/collect/ingestion_methods/sekoiaio_docker_concentrator.md

```diff
@@ -0,0 +1,435 @@
+    ```bash
+    sudo apt install telnet && telnet intake.sekoia.io 10514
+    ```
+
+    This command should return the following answer, meaning the network flow is opened:
+
+    ```bash
+    Connected to intake.sekoia.io.
+    Escape character is '^]'.
+    ```
+
+    Remove `telnet` if don't need it anymore
+
+    ```bash
+    sudo apt remove telnet
+    ```
+
+3. Finally check the status of the SEKOIA.IO plateform on [https://status.sekoia.io](https://status.sekoia.io).
+
+## Additional information
+
+The image used to run the concentrator is maintained on [this github repository](https://github.com/SEKOIA-IO/sekoiaio-docker-concentrator). Feel free to contribute and make a pull request to improve the concentrator!
+
+## How to update the concentrator
+Docker uses the notion of tag to identify the version of an image. The tag is always referenced in line starting with `image` in `docker-compose.yml`:
+
+```
+image: ghcr.io/sekoia-io/sekoiaio-docker-concentrator:0.9
+```
+
+`0.9` means the version used by `docker compose` is 0.9. You can find all the versions available on the GitHub repository [here](https://github.com/SEKOIA-IO/sekoiaio-docker-concentrator/pkgs/container/sekoiaio-docker-concentrator/versions?filters%5Bversion_type%5D=tagged)
+
+To update the concentrator, just change the tag in `docker-compose.yml`, then recreate the concentrator with the command:
+```bash
+sudo docker compose up -d
+```
+
+There is a special tag called `latest` which always refers to the latest version. If you use this tag, every time the concentrator will be recreated, the image will be updated with the latest version.
+
+!!! Important
+    Be very careful if you use the tag `latest` in production! If a major change is present in a new version, it could break your current concentrator with the settings in place.
+
+!!! Important
+    Each image version is also updated every week in order to update the OS contained in the image with the latest security patches. You are responsible for recreating the concentrator periodically to initiate the image update.
+
+## How do I add a new Intake later ?
+To add a new Intake, it's very simple ! Follow these steps:
+
+1. Edit the `intakes.yaml` file and add the new intake
+    ```yaml
+    - name: NewIntake
+      protocol: tcp
+      port: 20519
+      intake_key: INTAKE_KEY_FOR_NEW_INTAKE
+    ```
+2. Edit the `docker-compose.yml` file and modify the `ports` section accordlingly to the `intakes.yaml` file
+    ```yaml
+    ports:
+    - "20516-20519:20516-20519"
+    ```
+3. Recreate the concentrator by running the command:
+    ```bash
+    sudo docker compose up -d
+    ```
+4. Check in the concentrator logs that the new Intake is taken into account
+    ```bash
+    sudo docker compose logs | more
+    ```
+
+## 5 minutes setup on Debian
+
+!!! Warning
+    This script will automate all the steps detailed on this page **for a Debian host**. Please carefully read the content of this page prior to execute it.

```

Comment:
This script will automate all the steps detailed on this page **for a Debian host**. Please read carefully the content of this page prior to execute it.